frontend/src/main/pages/HomePage.js

```diff
@@ -20,11 +20,7 @@ export default function HomePage() {
   });
 
   const onSuccess = (courses) => {
-    //Toast only in place while Table component has not been added to the page
-    //The toast helps us test that the correct input has been received
-    //After the table component has been added, we can directly check whether the table has the received result
-    toast(courses.classes.length + " Courses Retrieved");
```

Comment:
A toast saying that courses were loaded might be nice, though not required

Reply:
Yea but we don't really need the toast because the users directly see the results in the table

Reply:
Fair enough